.github/actions/lint/Invoke-NodeJsLinter.ps1

```diff
@@ -46,6 +50,28 @@ function Install-NodeJsPackage($LibraryPath)
 if ($Paths.Trim())
 {
     $startPath = $PWD.Path
+
+    if ($Paths.Trim().StartsWith('{'))
+    {
+        $pathItems = $Paths | ConvertFrom-Json -AsHashtable
+        $pathItems.Keys | ForEach-Object {
+            $projectPath = (Get-Item $PSItem).FullName
+            $sourcePath = Join-Path $LibraryPath 'config' 'consumer' 'package.project.json'
+            $targetPath = Join-Path $projectPath 'package.json'
+
+            # In this case we assume that there is no project.json file, otherwise the config would already be in it.
```

Comment:
Let's test for that, though, with a hard failure.

Reply:
Why is that necessary?
Also with hard failure, if you lint different things (e.g. JS and CSS) for the same project, then the generated package.json from the first operation would trip the second operation. I would have to create additional cleanup logic which otherwise would not be needed.

Reply:
OK, leave it like that.